docs/references/subcommands/report.md

```diff
@@ -24,6 +24,41 @@ fossa report attribution --timeout 60
 
 Where `60` is the maximum number of seconds to wait for the report to be downloaded.
 
+### Valid Report Targets
+
+#### Project Directory
+
+You can specify a project directly like you would with `fossa analyze` to generate a report.
+For example:
+
+```
+fossa report attribution --format json ~/my-project
+```
+
+With no final path, FOSSA will try to fetch a report for the current directory's project.
+
+#### SBOM Files
+
+You can specify an SBOM that you would like to generate a report for by listing its filename:
+
+```
+fossa report attribution --format json ~/my-project-sbom.txt
+```
```

Comment:
This only retrieves the existing report if this SBOM has previously been analyzed, correct? If so please update the docs as this makes it sound like this will also analyze the SBOM on demand if needed

Reply:
Good catch, I've updated it.